### PR TITLE
Configuration for keyvault in unified pipelines.

### PR DESCRIPTION
**Pull request by mitchdenny**

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Key Vault Certificates client library</Description>
     <AssemblyTitle>Microsoft Azure.Security.KeyVault.Certificates client library</AssemblyTitle>
-    <VersionPrefix>4.0.0</VersionPrefix>
+    <Version>4.0.0</Version>
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj]: At this point in time given the instructions @weshaggard sent out, shouldn't this be 4.0.0-preview.6? (Our last shipped preview was preview.5.)

File: sdk/keyvault/ci.yml [hunk 2]
```diff
@@ -21,18 +26,22 @@ pr:
...
   parameters:
     ServiceDirectory: keyvault
+    ArtifactName: packages
+    Artifacts:
+    - name: Azure.Security.KeyVault.Certificates
+      safeName: AzureSecurityKeyVaultCertificates
+    - name: Azure.Security.KeyVault.Keys
+      safeName: AzureSecurityKeyVaultKeys
+    - name: Azure.Security.KeyVault.Secrets
+      safeName: AzureSecurityKeyVaultSecrets
```
Comment (heaths) [inline sdk/keyvault/ci.yml]: Nit: the .editorconfig dictates to add new lines to EOF, though probably doesn't cover .yml. Would be nice to keep a new line at the end here to help with diff'ing/merging in the future.

**Review by heaths — APPROVED** [review]

File: sdk/keyvault/ci.yml (hunk 2 above)
Comment (mitchdenny) [reply sdk/keyvault/ci.yml]: Fixed.

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj (hunk 1 above)
Comment (mitchdenny) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj]: Fixed for now -- but I think it'll be going to 4.0.0 very shortly.

**Pull request merged**